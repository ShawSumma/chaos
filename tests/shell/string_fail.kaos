str k = 'chaos'
k[0] = 5
k[0] = 'asdasdasd'
k

k[1] = 'k'
del k[0]
print k
