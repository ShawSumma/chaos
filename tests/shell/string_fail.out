Chaos Error:
Module: ~/chaos/__interactive__.kaos", line 2, in <module>
Illegal character assignment for string: k
Absorbed by Interactive Shell
Chaos Error:
Module: ~/chaos/__interactive__.kaos", line 0, in <module>
Right hand side of the assignment is not a character for string: k
Absorbed by Interactive Shell
chaos
kaos
